app/tests/e2e.raml.test.ts

```diff
@@ -0,0 +1,190 @@
+import fetch from "node-fetch";
+import {
+  HttpMethod,
+  baseUrl,
+  testId,
+  transformedTestId,
+} from "../../fixtures/utils/test-utils";
+
+const contentBody = (method: HttpMethod) =>
+  JSON.stringify({ message: `Hello world ${method}` });
+
+const httpRequest = async <T extends { message: string }>({
+  method,
+  content,
+  request = baseUrl,
+}: {
+  method: HttpMethod;
+  content?: string;
+  request?: string;
+}): Promise<{ status: number; message?: T["message"] }> => {
+  const response = await fetch(request, {
+    method,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: content,
+  });
+
+  const body =
```

Comment:
I think that this behaviour would have been better represented as assertions rather than switching based on the response code. Consider this for the future.